scripts/delayed.js

```diff
@@ -106,6 +106,53 @@ function loadBottomAd() {
   }
 }
 
+function makeSticky() {
+  const adWrappers = document.querySelectorAll('.ad-wrapper');
+  const lastAd = adWrappers[adWrappers.length - 1];
+
+  const bannerWrappers = document.querySelectorAll('.banner-wrapper');
+  const lastBanner = bannerWrappers[bannerWrappers.length - 1];
+
+  // Calculate the initial position of the elements
+  const lastAdOffsetTop = lastAd.offsetTop;
+
+  function handleScroll() {
+    const { scrollY } = window;
+    const screenWidth = window.innerWidth;
+
+    const mobileScreenWidthThreshold = 768;
+
+    // Check if the user has scrolled past the last ad,
+    // the screen width is greater than the threshold,
+    // and the scroll position is within the bounds of maxScrollPosition
+    if (
+      scrollY >= lastAdOffsetTop
+      && screenWidth > mobileScreenWidthThreshold
+    ) {
+      // Make the last ad and banner sticky
+      lastAd.style.position = 'fixed';
+      lastAd.style.top = '0';
+      lastAd.style.width = '330px';
+
+      lastBanner.style.position = 'fixed';
+      lastBanner.style.top = '300px';
+      lastBanner.style.width = '330px';
+    } else {
+      // Reset the style when scrolling up, on mobile, or past the maxScrollPosition
+      lastAd.style.position = 'static';
+      lastAd.style.top = 'auto';
+      lastAd.style.width = 'auto';
+
+      lastBanner.style.position = 'static';
+      lastBanner.style.top = 'auto';
+      lastBanner.style.width = 'auto';
+    }
+  }
```

Comment:
would it make sense to toggle the classes instead of adding it all in the js plus adding some improvements around debouncing the scroll event so it isnt firing like crazy every second user scrolls the page, something like below

 ```
 function handleScroll() {
    clearTimeout(scrollTimeout);
    scrollTimeout = setTimeout(() => {
      const { scrollY } = window;
      const screenWidth = window.innerWidth;
      const shouldSticky = scrollY >= lastAdOffsetTop && screenWidth > mobileScreenWidthThreshold;

      lastAd.classList.toggle('sticky', shouldSticky);
      lastBanner.classList.toggle('sticky', shouldSticky);
    }, 50); 
  }
```